apps/bfd-server/bfd-server-war/src/main/resources/security_labels.yml

```
@@ -133,15 +133,18 @@
 - system: https://www.cms.gov/Medicare/Coding/HCPCSReleaseCodeSets
   code: G2070
   startDate: 1970-01-01
-  endDate: ACTIVE  
+  endDate: ACTIVE
+  comment: No longer active as of 01-01-2025
```

Sorry, not trying to be picky here, but I think we want to convey why they're being kept active so somebody who looks at this a year from now won't be confused.

```suggestion
  comment: No longer active as of 01-01-2025. These are being retained temporarily in case of accidental use.
```